client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1913,6 +1914,16 @@ public void testGetTemplateRequest() throws Exception {
         assertThat(request.getEntity(), nullValue());
     }
 
+    public void testGetStoredScriptRequest() {
+        GetStoredScriptRequest storedScriptRequest = new GetStoredScriptRequest("x-script");
+        Map<String, String> expectedParams = new HashMap<>();
+
+        Request request = RequestConverters.getStoredScript(storedScriptRequest);
+        assertThat(request.getEndpoint(), equalTo("/_scripts/" + storedScriptRequest.id()));
+        assertThat(request.getParameters(), equalTo(expectedParams));
```

Comment:
no need to create a new map right? you can just assert that parameters is empty.